Makefile

```diff
@@ -155,6 +155,7 @@ operator-image: install caas/jujud-operator-dockerfile caas/jujud-operator-requi
 	cp caas/jujud-operator-dockerfile ${JUJUD_STAGING_DIR}
 	cp caas/jujud-operator-requirements.txt ${JUJUD_STAGING_DIR}
 	docker build -f ${JUJUD_STAGING_DIR}/jujud-operator-dockerfile -t ${DOCKER_USERNAME}/caas-jujud-operator:${OPERATOR_IMAGE_TAG} ${JUJUD_STAGING_DIR}
+	rm -rf ${JUJUD_STAGING_DIR}
```

Comment:
should this remove the dir or the contents of the dir? (i.e. rm -fr ${JUJUD_STAGING_DIR/*}?

Reply:
the whole dir - we mkdir the dir above
normally it lives in /tmp but if i override it to be somewhere else like ~ i don't want i hanging around

Reply:
ack, I should have extended the context and seen that. LGTM